tests/monotouch-test/Foundation/FileManagerTest.cs

```diff
@@ -37,6 +37,23 @@ public class NSFileManagerTest {
 			}
 		}
 
+		// we might believe that Envioment.UserName os the same as NSFileManager.UserName, but it is not. On the simulator for
+		// example, NSFileManager.UserName is an empty string while mono returns 'somebody'
+		[Test]
+		public void GetUserNameTest () => Assert.IsNotNull (NSFileManager.UserName);
```

Comment:
Cute using arrows here, but I'd suggest moving to a block to make stepping a bit easier if we need to debug (?)

Reply:
In the test? No need